tests/specs/functional/infinite-gas-spec.k

```diff
@@ -20,124 +20,111 @@ module INFINITE-GAS-SPEC
  // Gas simplifications
  // -------------------
 
-    claim <k> runLemma(#if B #then (VGas -Int Csstore(ISTANBUL, V1, V2, V3)) +Int -435 #else (VGas -Int Csstore(ISTANBUL, V1, V2, V3)) +Int -4342 #fi)
+    claim [gas.arith.0]: <k> runLemma(#if B #then (VGas -Int Csstore(ISTANBUL, V1, V2, V3)) +Int -435 #else (VGas -Int Csstore(ISTANBUL, V1, V2, V3)) +Int -4342 #fi)
```

Comment:
Should I remove the names now they are passing? Or is it better to keep them around now they are there?

Reply:
Keep them please.